### DISCUSSION TITLE
Collapsible list

### DISCUSSION THREAD

**Discussion by Muad0510** [GitbookIO/community#223] [Feature Requests]
### What's your feature/idea?

Right now collapsible blocks and lists are two separate features, with no way to merge them into one in an easy and aesthetic way.

Collapsible lists that act as a combination of an unordered list and a collapsible block.

### What's your desired outcome?

Have a list option where the hierarchy is collapsible, sub points/levels can be hidden under higher levels.

Exactly how markdown handles it with, but with multiple levels and the functionality and hierarchy of a list.
`<details><summary>Hello</summary><blockquote>
  <details><summary>World</summary><blockquote>
    :smile:
  </blockquote></details>
</blockquote></details>`

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Thanks for your suggestion! 

Note: This might be related to https://github.com/orgs/GitbookIO/discussions/16

**Reply by Muad0510** [reply]
Somewhat related. My request is more for being able to nest expandable blocks inside each other, plus having a list where each level can be collapsed, hiding lower level content..
